entity-service/src/main/kotlin/com/egm/stellio/entity/authorization/EntityAccessControl.kt

```diff
@@ -0,0 +1,57 @@
+package com.egm.stellio.entity.authorization
+
+import com.egm.stellio.shared.util.AccessRight
+import com.egm.stellio.shared.util.AuthContextModel
+import com.egm.stellio.shared.util.AuthContextModel.AUTH_PROP_RIGHT
+import com.egm.stellio.shared.util.JsonLdUtils
+import com.egm.stellio.shared.util.toNgsiLdFormat
+import java.net.URI
+import java.time.ZonedDateTime
+
+data class EntityAccessControl(
+    val id: URI,
+    val type: List<String>,
+    val createdAt: ZonedDateTime,
+    var modifiedAt: ZonedDateTime? = null,
+    val right: AccessRight,
+    val specificAccessPolicy: AuthContextModel.SpecificAccessPolicy? = null,
+    var contexts: List<String> = mutableListOf()
```

Comment:
it seems it is never set... and I don't really see how it could be set. remind me to talk about it today.